Uno.Gallery/Uno.Gallery.Wasm/FragmentNavigationHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Uno.Foundation;
+
+namespace Uno.Gallery.Wasm
+{
+	public class FragmentNavigationHandler
+	{
+		public static string CurrentFragment
+		{
+			get
+			{
+				const string command = "Uno.Gallery.Wasm.FragmentNavigationHandler.getCurrentFragment()";
+				var fragment = WebAssemblyRuntime.InvokeJS(command);
+				return fragment;
+			}
+
+			set
+			{
+				var escaped = WebAssemblyRuntime.EscapeJs(value);
+				var command =
+					"Uno.Gallery.Wasm.FragmentNavigationHandler.setCurrentFragment(\"" + escaped + "\")";
+
+				WebAssemblyRuntime.InvokeJS(command);
+			}
+		}
+	}
```

Comment:
Let's use JSImport instead, escapeJS is not performant and will be removed in the future.

Reply:
Done. Updated to use JSImport.

https://github.com/unoplatform/Uno.Gallery/pull/1054/commits/82d1b5bbe40084998cb84cf075c638c3fef35e4f